### DISCUSSION TITLE
Cloud Variables and Local Storage

### DISCUSSION THREAD

**Discussion by ejnalo** [TurboWarp/packager#715] [Ideas]
I suggest adding some options which remove some cloud variables limits such as:
- Allowing letters
- Removing variable length limit

But these options will be available if the cloud host is set to localStorage (or maybe to a custom server)

I already know that cloud variables limit (10) can be overridden when packaging

**Comment by CST1229** [comment]
I'm pretty sure cloud variables can have infinite length and letters when stored in local storage, and the limits on other servers depends on the server, not client.
